dbcon/mysql/ha_mcs_opt_rewrites.cpp

```diff
@@ -73,38 +73,52 @@ void in_subselect_rewrite_walk(const Item* item_arg, void* arg)
     }
 }
 
-/* @brief  opt_flag_unset_PS() - Unsets first_cond_optimization */
+// Sets SELECT_LEX::first_cond_optimization
+void flag_set(SELECT_LEX* select_lex)
```

Comment:
Reading the name of the function I can't get what is the flag that we manipulate with. Could you somehow make the functions names more cluefull?

Reply:
Done.